languages/nim/exercises/concept/basics/.meta/design.md

```diff
@@ -0,0 +1,22 @@
+## Learning objectives
+
+- Know about `proc` definition
+- Know `const` definition
+- Know about type inference
+- Know about `proc` parameters
+- Know the different way of returning values from a `proc` (implicit, `return`, `result`)
+- Know how to export identifiers
+- Know how to print values for debugging (`echo`, `debugEcho`)
+
+## Out of scope
+
+- `func` (I think we'll do another exercise on this so don't what to put here)
+- `var`, `let`, `var` assignment
```

Comment:
```suggestion
- `var`, `let` assignment
```

Reply:
One thing to consider is that any subsequent exercise won't be able to use `var` or `let`, unless they depend on another exercise in which these concepts are introduced. This is totally valid of course, but is something to consider.

Reply:
It's not so important, that could even be the next exercise.
`let` is immutable and therefore can not be assigned to.
`var` is mutable and therefore there is the initialization of the variable and the assignment of the variable afterward.

Reply:
> It's not so important, that could even be the next exercise.

Sounds good